packages/iTwinUI-react/src/core/utils/hooks/useDragAndDrop.tsx

```diff
@@ -68,9 +51,9 @@ export const useDragAndDrop = (
       left,
     } = elementRef.current?.getBoundingClientRect();
 
-    let newTranslateX = translateX.current;
-    let newTranslateY = translateY.current;
+    let [newTranslateX, newTranslateY] = getTranslateValues(elementRef.current);
```

Comment:
Getting `translate` value from the element and not from the `translateX.current` because when resizing, it triggers resize hook inside `useDragAndDrop` and it will use old translate values that breaks the resizing.